vti/src/cli.ts

```diff
@@ -165,6 +165,6 @@ async function getDiagnostics(workspaceUri: URI) {
     console.log('  vti diagnostics ---- Print all diagnostics');
     console.log('');
   }
-})().catch(_err => {
-  console.error('VTI operation failed');
+})().catch(err => {
+  console.error(`VTI operation failed with error: ${err}`);
```

Comment:
I think it should use `process.exit(1);` after this
```suggestion
  console.error(`VTI operation failed with error: ${err}`);
  process.exit(1);
```